text/first-push.md

```diff
@@ -11,7 +11,7 @@ impl List {
 }
 ```
 
-First thing's first, we need to make a node to store our element in:
+First things first, we need to make a node to store our element in:
```

Comment:
Hmm... I thought this was correct "first thing's first" -> "first thing is first"